private/buf/cmd/buf/command/alpha/plugin/pluginpush/pluginpush.go

```diff
@@ -110,6 +118,11 @@ func run(
 	if err := bufcli.ValidateErrorFormatFlag(flags.ErrorFormat, errorFormatFlagName); err != nil {
 		return err
 	}
+	if len(flags.OverrideRemote) > 0 {
+		if _, err := netextended.ValidateHostname(flags.OverrideRemote); err != nil {
```

Comment:
This feels like good validation to perform on this arg - I'm sure Docker performs similar validation on the remote.